files/templates/alertmanager/alertmanager.yaml

```diff
@@ -507,6 +507,12 @@ inhibit_rules:
     - cancel_if_cluster_is_not_running_prometheus_agent=true
   equal: [cluster_id]
 
+- source_matchers:
+    - inhibition_prometheus_agent_down=true
```

Comment:
I'm confused, the new doc says it should be inhibit_prometheus_agent_down https://github.com/giantswarm/prometheus-rules/pull/709/files